proto/astarteplatform/msghub/property.proto

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file is part of Astarte.
+ *
+ * Copyright 2024 SECO Mind Srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+syntax = "proto3";
+
+package astarteplatform.msghub;
+
+import "astarteplatform/msghub/astarte_type.proto";
+import "astarteplatform/msghub/interface.proto";
+
+/* A message representing the property value associated to a certain interface and path. */
+message Property {
+  string path = 1;                      // Property path.
+  AstarteDataTypeIndividual value = 2;  // Astarte data.
+}
+
+/* A message representing the property value associated to a certain interface and path.
+ * Required for the `GetProperty` method that could need to return an unset property.
+ */
+message OptionalProperty {
```

Comment:
Maybe this could be called IndividualProperty, but it's just a nitpick, nothing against OptionalProp.

Other than that LGTM